src/Projekt/Projekt.fsproj

```diff
@@ -81,16 +57,54 @@
   <Target Name="BeforeBuild">
   </Target>
   -->
-  <ItemGroup>
-    <Templates Include="$(SolutionDir)\templates\**\*.*"/>
-  </ItemGroup>
   <Target Name="AfterBuild">
-    <Copy
-        SourceFiles="@(Templates)"
-        DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)"
-        />
+    <Copy SourceFiles="@(Templates)" DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)" />
```

Comment:
Yes, I had that problem with VS as well, but I don't normally use VS! I just tried splitting it into a separate file, so create a new `Template.targets`:

```
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
<ItemGroup>
  <Templates Include="$(SolutionDir)\templates\**\*.*" />
</ItemGroup>
</Project>
```

and use `<Import Project="Template.targets" />`. Does that trick VS?